ios/src/ViewController.mm

```diff
@@ -201,16 +200,18 @@ - (void)setupGL
 {
     [EAGLContext setCurrentContext:self.context];
 
-    Tangram::initialize("scene.yaml");
-    Tangram::loadSceneAsync("scene.yaml");
-    Tangram::setupGL();
+    if (!self.map) {
+        self.map = new Tangram::Map("scene.yaml");
```

Comment:
Yeah, not too worried about it because we would only ever delete it when the process is going to exit anyway - but it's probably better practice to explicitly manage it.